week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,44 +1,69 @@
+// Fix Functions
+
+// Aim: to understand the change code inside functions
+//
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
+// 1) mood function does this return `"I am not happy"`
+//    Only make edits inside the function
+
 function mood() {
   let isHappy = true;
 
   if (isHappy) {
-    return "I am happy";
-  } else {
     return "I am not happy";
+  } else {
+    return "I am  happy";
   }
 }
+console.log(mood());
+// 2) For any numerical value greater or equal to 10
+//    Hint: use constant `num` and only change isBigEnough.
+//    Variable isBigEnough needs to evaluate to a boolean
 
 function greaterThan10() {
-  let num = 10;
-  let isBigEnough;
+  const num = 10;
+  const isBigEnough = true;
```

Comment:
Here you have wrong, isBigEnough should isBigEnough = num >= 20 or 30; then the if else statement return the message if the isBigEnough is true or false based on the variable which true or false with the comparaison operator. Google true or false comparaison operator.